prime-router/docs/design/proposals/translation-template-test-plan.md

```diff
@@ -0,0 +1,47 @@
+# Translation Template Test Plan
+
+## Problem Statement
+As a Reportstream engineer, I want tests that verify that the changes I make to translations will not break essential 
+rules set forth by the spec or our users. 
+
+Currently, we use integration tests for this purpose, but these do not give us the ability to easily tell which thing 
+broke/changed or even really what is specifically being tested.
+
+## Considerations
+- We not only want to check the rules set forth in the spec, but the ones set forth by our users.
+
+## Proposal on *what* to test
```

Comment:
I would maybe also include that we have tests for each segment/FHIR resource, i.e. we can losslessly go from `MSH->MessageHeader->MSH` for every component in MSH